components/title/multisize/src/index.scss

```diff
@@ -0,0 +1,60 @@
+@import '~@schibstedspain/theme-basic/lib/index';
+
+$c-title-multisize-primary: #a81c4f !default;
```

Comment:
Ok, I am changing the styles variables definitions in order to use the theme-basic colors defined.